librz/core/cmd/cmd_analysis.c

```diff
@@ -9138,4 +9119,28 @@ RZ_IPI RzCmdStatus rz_analyse_name_handler(RzCore *core, int argc, const char **
 		return RZ_CMD_STATUS_ERROR;
 	}
 	return RZ_CMD_STATUS_OK;
-}
\ No newline at end of file
+}
+
+RZ_IPI RzCmdStatus rz_analysis_all_esil_handler(RzCore *core, int argc, const char **argv) {
+	bool reg_flags_defined = !!rz_flag_space_count(core->flags, RZ_FLAGS_FS_REGISTERS);
```

Comment:
```suggestion
	bool reg_flags_defined = rz_flag_space_count(core->flags, RZ_FLAGS_FS_REGISTERS) != 0;
```
is more readable IMO?